cmd/utils/cmd.go

```diff
@@ -387,9 +387,10 @@ type exportHeader struct {
 }
 
 const exportMagic = "gethdbdump"
+
 const (
-	OpBatchAdd = 0
-	OpBatchDel = 1
+	OpBatchAdd = iota
+	OpBatchDel
```

Comment:
Not much point in iota:ing something which is only two items. I mean, it's not _wrong_, but hardly enough gains to justify a change at all.